classes/Commentions.php

```diff
@@ -172,12 +172,20 @@ public static function sanitize($data, $update = false)
 
         // timestamp is required; use current time if missing or not a unix epoch
         if (! empty($data['timestamp']) && ! is_numeric($data['timestamp'])) {
```

Comment:
Why are you testing against if the timestamp is numeric value? Shouldn’t the timestamp be a string value (in `Y-m-d H:i` format) value or am I getting something wrong?

Reply:
That didn't make sense indeed (and was actually the cause of a bug I had been observing yesterday). The update to the PR should fix this.